guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,18 +1,63 @@
 [id="Resolving_Package_Dependencies_{context}"]
 = Resolving Package Dependencies
 
+.Introduction
+
```

Comment:
```suggestion
```

IMHO not necessary because a) all concepts should start with an introduction and b) there's no text between this heading and the level 1 heading above.